### ISSUE TITLE
Passive jammer failed to start

### ISSUE THREAD

**Issue by octavo** [kaputnikGo/PilferShushJammer#50]
![img_9.jpg](https://user-images.githubusercontent.com/1730033/196502099-e9be8fd8-5a04-412c-82b3-84d38650a54c.jpg)

**Comment by kaputnikGo** [comment]
that means that another user app has hold of, or is using, the microphone. identifying it can be tricky due to the later versions of android allowing concurrent use of the microphone with certain conditions.

**Comment by octavo** [comment]
How do I find out?

On my phone, no apps are allowed to use it all the time; the camera and sms apps are allowed only while in use; and the communication apps ask every time.

**Comment by kaputnikGo** [comment]
using the pifershush app you can go to the Inspector tab and find a list user apps that have microphone access. this will give you an idea about which one may be using it. Selecting one of the apps listed will show the summary of that app and you can see the list of capabilities, including record_audio, and the list of services and receivers. It may be that an app is using a background service to periodically use the microphone for something.  Finding which app it is can be tricky.

**Comment by octavo** [comment]
If I uninstall all the listed apps will the jammer work?

**Comment by kaputnikGo** [comment]
i cant say for sure and i wouldnt recommend uninstalling all the apps. try to identify a suspect app, depending on your android version you may be able to revoke permissions and see if it helps. it could also be a system app or voice assistant app.

**Comment by octavo** [comment]
I uninstalled all the listed apps apart from pilfersush to no avail.

**Comment by kaputnikGo** [comment]
yes it could be a system app or assistant app. check settings and the list of assistant apps. if you have android 10 or above they allow concurrent audio recording and voice assistants to dominate the mic.

**Comment by octavo** [comment]
Which apps should I look for?

**Comment by kaputnikGo** [comment]
any that have record audio capability
